skyportal/handlers/api/photometry.py

```diff
@@ -664,6 +665,8 @@ def get_values_table_and_condition(df):
     df: `pandas.DataFrame`
         Dataframe with the columns 'obj_id', 'instrument_id', 'origin',
         'mjd', 'standardized_fluxerr', 'standardized_flux'.
+    ignore_flux: bool
```

Comment:
```suggestion
    ignore_flux : bool
```

Reply:
Why is that space needed for this argument but not the others?

Reply:
It should be everywhere, according to the numpydoc standard.